### ISSUE TITLE
Passive mode...

### ISSUE THREAD

**Issue by pjgreen89** [DankMemer/dank-memer-suggestions-and-bugs#177]
**Is your feature request related to a problem? Please describe.**
It is a very frustrating thing when you’re trying to rob people who just sit in passive mode all the time and don’t actually do anything ... Spending ages typing out multiple usernames to find out 99% are passive ... 

**Describe the solution you'd like**
I think passive mode should be limited to 5 days with a 2 day cool down period where they can’t enter passive mode until the 3rd day. Or if they want to remain in passive they have to sacrifice their wallet. Would encourage people to actually use the bot more and actually think properly 
**Describe alternatives you've considered**
Alternatively people in passive mode can’t gamble either. Taking a lot away from the bot when someone can make millions and have no worries about anything... it’s almost like playing FIFA against a 5 year old.

**Additional context**
In summary make passive mode a timed setting 5 days on ... 2 days off ... or after 5 days to stay in passive they lose some/most of their wallet

**Comment by pjgreen89** [comment]
Also, what is the point in fake id’s, padlocks, landmines when every is going passive ...

**Comment by RandomTurtleBoi** [comment]
If you want to rob from people, just do `pls rich` and the people with a blue dash by their name aren't passive. There are multiple pages of the rich list, so you're pretty much guaranteed to find someone out of passive.

**Comment by pjgreen89** [comment]
I know that much... But people stay passive and it’s taking a lot away from the bot. They’re in passive and it’s like oh I’ll gamble and be safe.
Do you think you’d consider making it a time based thing to keep people on their toes?


Sent from my iPhone

On 2 May 2020, at 14:59, RandomTurtleBoi <notifications@github.com> wrote:

﻿

If you want to rob from people, just do pls rich and the people with a blue dash by their name aren't passive. There are multiple pages of the rich list, so you're pretty much guaranteed to find someone out of passive.

**Comment by RandomTurtleBoi** [comment]
Probably not, I use passive because I happen to have a lot of money in my wallet that I don't want to lose (plus, I don't like stealing myself). 'oh I’ll gamble and be safe' Passive mode removes 25% of your winnings and alcohol doesn't boost gambling anymore. Witn inactive users, _maybe_ they're just taking a break from the bot and will eventually come back. They can use passive to ensure the safety of their coins. They could just be taking a break, or offline for whatever reason. There is a 24 hour cool down to turn passive mode back on when they turn it off, so that's a potential cooldown period you could use.

**Comment by pjgreen89** [comment]
True... But what’s the point of padlocks and landmines if everyone’s passive?
I understand the frustration when losing coins, I lose more coins than anyone gambling... 🤣
However it’s the people who come on every day, in passive mode, rack up hundreds of thousands and just sit there ... it does take a bit of excitement/annoyance out of it...
it would also make things more nerve wracking ... knowing you’re coming out of passive with 2 days as a target so they buy landmines and padlocks...

I just think it would be a good addition lol


Sent from my iPhone

On 2 May 2020, at 21:06, RandomTurtleBoi <notifications@github.com> wrote:

﻿

Probably not, I use passive because I happen to have a lot of money in my wallet that I don't want to lose (plus, I don't like stealing myself). 'oh I’ll gamble and be safe' Passive mode removes 25% of your winnings and alcohol doesn't boost gambling anymore. Witn inactive users, maybe they're just taking a break from the bot and will eventually come back. They can use passive to ensure the safety of their coins. They could just be taking a break, or offline for whatever reason. There is a 24 hour cool down to turn passive mode back on when they turn it off, so that's a potential cooldown period you could use.

**Comment by melmsie** [comment]
will be addressed later

**Comment by pjgreen89** [comment]
I’m glad it’s being addressed and thought about because I think people just sitting in passive mode all the time is taking the fun out of the bot

Sent from my iPhone

On 3 May 2020, at 01:03, Austin Melms <notifications@github.com> wrote:

﻿

will be addressed later
